packages/lab/src/toolbar/Toolbar.css

```diff
@@ -1,7 +1,7 @@
 .uitkToolbar {
   --toolbar-background: var(--uitk-container-background-high);
   --toolbar-fontSize: var(--uitk-text-fontSize);
-  /* --uitkFormActivationIndicator-size: 0; */
+  /* --uitkFormField-activationIndicator-size: 0; */
```

Comment:
Should maybe just remove this if it's commented out?

Reply:
I think the comments need to stay because the styling hasn't been done yet and I believe were left by Steve to probably help whoever comes round to styling it!